packages/default/scss/skeleton/_layout.scss

```diff
@@ -0,0 +1,65 @@
+@include exports("skeleton/layout") {
+
+    .k-skeleton-grid {
+        padding: .5em 0;
+        margin: .5em 0;
+        display: inline-grid;
+        grid-template-columns: repeat(10, 1em);
+        grid-template-rows: repeat(10, 1em);
```

Comment:
Hardcoding like this is not okay. I believe this should look like this:

```
$skeleton-grid-columns-count: 10 !default;
$skeleton-grid-columns-size: 1fr !default;

grid-template-columns: repeat($skeleton-grid-columns-count, $skeleton-grid-columns-size);
```

I understand why you did that - the `.k-skeleton-circle`, however, you can handle this scenario like the badge - 

https://github.com/telerik/kendo-themes/blob/develop/packages/default/scss/badge/_variables.scss#L23
https://github.com/telerik/kendo-themes/blob/develop/packages/default/scss/badge/_layout.scss#L48